implemented SSSDK (#4)
deveronuas/sssdk: README.md

# Simple Salesforce SDK

> SSSDK is a simple framework that can be used as a replacement of the Salesforce's connected mobile SDK application.SSSDK supports iOS 15 or above(SwiftUI).  

Reviewer: ```suggestion
This is a simplified SDK that can be used as a replacement for Salesforce's connected mobile SDK in your SwiftUI application. It supports iOS 15 or above.
```

```

### login
Returns login view with the configured host and redirectUri 

> (for more information please visit the following link -> https://help.salesforce.com/s/articleView?id=sf.remoteaccess_oauth_user_agent_flow.htm&type=5)

Reviewer: If you would like to refer docs, maybe better if refer Salesforce docs?

```

### refershAccessToken
Refreshes the `access_token`, then updates the keychain.

> (for more information please visit the following link -> https://datatracker.ietf.org/doc/html/rfc6749#section-6) 

Reviewer: I'm not sure if this adds much value (given that we have created the SDK to abstract this from the user)

Might be place these docs in the code base for some who is looking to see how this works.

```
SSSDK.shared.fetchData(by query:)
```
## Installation

#### Swift Package Manager

SSSDK is available through [Swift Package Manager](https://swift.org/package-manager/).

+ For App integration

#### Swift Package Manager
You can use [Swift Package Manager](https://swift.org/package-manager/) to install SwiftKeychainWrapper using Xcode:

> 1. Open your project in Xcode
> 
> 2. Click "File" -> "Swift Packages" -> "Add Package Dependency..."
> 
> 3. Paste the following URL: https://github.com/deveronuas/sssdk.git
> 
> 4. Click "Next" -> "Next" -> "Finish"

Reviewer: ```suggestion 
1. Open your project in Xcode
2. Click "File" -> "Add Packages..."
3. Paste the following URL: https://github.com/deveronuas/sssdk.git
4. Choose a `Dependency Rule` and finish
```

## General Usage

> import SwiftUI

Reviewer: Please use a swift code block

```suggestion

```swift
import SwiftUI
```

import SimpleSalesforceSDK

struct OAuthView: View {

>   let host = "your host url"
>   let clientId = "your client's id"
>   let clientSecret = "your client's secret"
>   let redirectUri = "your redirect uri"

Reviewer: Let's use some convention for constants, like this?

```suggestion
  let HOST = "your host url"
  let CLIENT_ID = "your client's id"
  let CLIENT_SECRET = "your client's secret"
  let REDIRECT_URI = "your redirect uri"
```

  let fetchQuery = "your SOQL query to fetch the data"

  @State private var launchLoginView = false
  init(){
    // Required -> To use the SSSDK framework you must configure it.
    SSSDK.shared.configure(host: host, redirectUri: redirectUri, clientId: clientId, clientSecret: clientSecret)
  }
  var body: some View {
    VStack{
      Button{
        launchLoginView.toggle()
      }label: {
        Text("Login")
      }
      .sheet(isPresented: $launchLoginView) {
        try? SSSDK.shared.login()

>           .onOpenURL(perform: { url in

Reviewer: not sure if this should be here... looks weird that this `onOpenURL` is attached to the login view

Author: I tried that we can use it anywhere but I can move it outside of that modal sheet.
